tests/SpecTests/TransactionsSpecTest.php

```diff
@@ -34,22 +35,17 @@ class TransactionsSpecTest extends FunctionalTestCase
      * @var array
      */
     private static $incompleteTests = [
-        'error-labels: add unknown commit label to MaxTimeMSExpired' => 'PHPC-1382',
-        'error-labels: add unknown commit label to writeConcernError MaxTimeMSExpired' => 'PHPC-1382',
-        'read-pref: default readPreference' => 'PHPLIB does not properly inherit readPreference for transactions',
-        'read-pref: primary readPreference' => 'PHPLIB does not properly inherit readPreference for transactions',
-        'run-command: run command with secondary read preference in client option and primary read preference in transaction options' => 'PHPLIB does not properly inherit readPreference for transactions',
-        'transaction-options: transaction options inherited from defaultTransactionOptions' => 'PHPC-1382',
-        'transaction-options: startTransaction options override defaults' => 'PHPC-1382',
-        'transaction-options: defaultTransactionOptions override client options' => 'PHPC-1382',
-        'transaction-options: transaction options inherited from client' => 'PHPLIB does not properly inherit readConcern for transactions',
-        'transaction-options: readConcern local in defaultTransactionOptions' => 'PHPLIB does not properly inherit readConcern for transactions',
-        'transaction-options: readConcern snapshot in startTransaction options' => 'PHPLIB does not properly inherit readConcern for transactions',
+        'transactions/read-pref: default readPreference' => 'PHPLIB does not properly inherit readPreference for transactions (PHPLIB-473)',
+        'transactions/read-pref: primary readPreference' => 'PHPLIB does not properly inherit readPreference for transactions (PHPLIB-473)',
+        'transactions/run-command: run command with secondary read preference in client option and primary read preference in transaction options' => 'PHPLIB does not properly inherit readPreference for transactions (PHPLIB-473)',
+        'transactions/transaction-options: transaction options inherited from client' => 'PHPLIB does not properly inherit readConcern for transactions (PHPLIB-473)',
+        'transactions/transaction-options: readConcern local in defaultTransactionOptions' => 'PHPLIB does not properly inherit readConcern for transactions (PHPLIB-473)',
+        'transactions/transaction-options: readConcern snapshot in startTransaction options' => 'PHPLIB does not properly inherit readConcern for transactions (PHPLIB-473)',
```

Comment:
I just realized that we missed scheduling [PHPLIB-473](https://jira.mongodb.org/browse/PHPLIB-473) for 1.5.0 since I skipped the triage meeting last Friday. I've updated the ticket so these skips can be investigated before the release, in case it's evidence of a lingering bug.